onnx/defs/nn/defs.cc

```diff
@@ -15,10 +15,18 @@ const char* pads_doc =
     "added at the beginning of axis `i` and xi_end, the number of pixels added at "
     "the end of axis `i`. This attribute cannot be used simultaneously with "
     "auto_pad attribute. If not present, the padding defaults to 0 along start and end of each spatial axis.";
-const char* auto_pad_doc =
+const char* conv_auto_pad_doc =
     "auto_pad must be either NOTSET, SAME_UPPER, SAME_LOWER or VALID. Where "
     "default value is NOTSET, which means explicit padding is used. "
-    "SAME_UPPER or SAME_LOWER mean pad the input so that the output spatial size match the input."
+    "SAME_UPPER or SAME_LOWER mean pad the input so that "
+    "`output_shape[i] = ceil(input_shape[i] / strides[i])` for each axis `i`. "
+    "In case of odd number add the extra padding at the end for SAME_UPPER and at the "
```

Comment:
Suggest changing the ```In case of odd number''' sentence as below for readability:
```
The padding is split between the two sides equally or almost equally (depending on whether it is even or odd).
In case the padding is an odd number, the extra padding is added at the end for SAME_UPPER
and at the beginning for SAME_LOWER.
```

Reply:
Same change below also